assets/xml/objects/gameplay_keep.xml

```diff
@@ -934,14 +934,14 @@
         <DList Name="gEffDustDL" Offset="0x23130" />
         <DList Name="gOwlStatueWhiteFlashDL" Offset="0x23210" />
         <DList Name="gameplay_keep_DL_023288" Offset="0x23288" />
```

Comment:
```suggestion
        <DList Name="gEffFlash1DL" Offset="0x23210" />
        <DList Name="gEffFlash2DL" Offset="0x23288" />
```
Since you're in this block, could you rename these? Just something to fix, `gOwlStatueWhiteFlashDL` is definitely not limited to `OwlStatue`, and both of these are identical to the `gEffFlash1DL` and `gEffFlash2DL` from OoT

I have the reservation to fix it, but I'll remove my name.

Also, `gLightOrbTex` -> `gFlashTex`